Makefile

```diff
@@ -1,13 +1,13 @@
-all             :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=200 dapp --use solc:0.6.12 build
-clean           :; dapp clean
+all             :; FOUNDRY_OPTIMIZER=true FOUNDRY_OPTIMIZER_RUNS=200 forge build --use solc:0.6.12
+clean           :; forge clean
 # usage:        make test match=Burn
 test            :; ./test.sh ${match}
 deploy          :; echo "use deploy-goerli or deploy-mainnet"
 deploy-goerli   :; make && ./scripts/deploy-goerli.sh
 deploy-mainnet  :; make && ./scripts/deploy-mainnet.sh
 flatten         :;
-	hevm flatten --source-file "src/UniswapV2Callee.sol" > out/UniswapV2CalleeDai.sol
-	hevm flatten --source-file "src/UniswapV2LpTokenCallee.sol" > out/UniswapV2LpTokenCalleeDai.sol
-	hevm flatten --source-file "src/UniswapV3Callee.sol" > out/UniswapV3Callee.sol
-	hevm flatten --source-file "src/WstETHCurveUniv3Callee.sol" > out/WstETHCurveUniv3Callee.sol
-	hevm flatten --source-file "src/CurveLpTokenUniv3Callee.sol" > out/CurveLpTokenUniv3Callee.sol
+	forge flatten "src/UniswapV2Callee.sol" > out/UniswapV2CalleeDai.sol
+	forge flatten "src/UniswapV2LpTokenCallee.sol" > out/UniswapV2LpTokenCalleeDai.sol
+	forge flatten "src/UniswapV3Callee.sol" > out/UniswapV3Callee.sol
```

Comment:
We need to change the output file name, or directory to `out/flatten` to prevent this conflict:

```bash
$ rm -fr out
$ make clean
forge clean
$ forge install
$ make
FOUNDRY_OPTIMIZER=true FOUNDRY_OPTIMIZER_RUNS=200 forge build --use solc:0.6.12
[⠊] Compiling...
[⠑] Compiling 86 files with 0.6.12
[⠰] Solc 0.6.12 finished in 5.84s
[SNIP ...]


$ make flatten
forge flatten "src/UniswapV2Callee.sol" > out/UniswapV2CalleeDai.sol
forge flatten "src/UniswapV2LpTokenCallee.sol" > out/UniswapV2LpTokenCalleeDai.sol
forge flatten "src/UniswapV3Callee.sol" > out/UniswapV3Callee.sol
/bin/sh: out/UniswapV3Callee.sol: Is a directory
make: *** [flatten] Error 1
```

Reply:
Addressed in #37